packages/webpack-loader/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ts-tools/webpack-loader",
-  "version": "1.1.0",
+  "version": "1.2.0",
```

Comment:
No need to change version... only I do it when I release (with `yarn lerna publish`, in the case of my repo)...